### PR TITLE
Ruby Tic Tac Toe: Game testing

### PR DESCRIPTION
**Pull request by Osong-Michael**
In this project we were asked to add specs for all public methods we created in the game, covering every edge cases.

### CODE REVIEW

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Awesome :+1: 
You did a commendable job with this PR. All your tests pass very well. However, we need to add more test cases. We need every test block to have a positive and a negative scenario.
- [ ] add a test case for the display_board public method. All public methods must be tested.
- [ ] check my comments for more suggestions

### Martin Afani

File: spec/games_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      expect(tictactoe.valid_input?('a')).to eql(false)
+      expect(tictactoe.valid_input?('@')).to eql(false)
+    end
+  end
+
+  describe '#update_board' do
+    it 'Updates board with user character if input is valid and position is available' do
+      tictactoe.update_board(player, 1)
+      expect(tictactoe.positions[1]).to eql('X')
+    end
```
Comment (whiz25) [inline spec/games_spec.rb:56]: - test what happens if input is invalid and position is unavailable. Does the board update?

**Comment by Temmarie** [comment]
We made the changes requested by the previous TSE which are : 
- adding a test case for all public methods and
- adding a negative test for update_board.

**Review by mosesogwo — APPROVED** [review]
**Hi team:clap: Good work:+1:. Project is approved**

Before you merge, add a negative test for your **#draw?** method just like you have done for **#update_board**

**Cheers:stars:**

### Status: Approved :heavy_check_mark:
